alpine-nix-rails-nginx/etc/nginx/static-content.conf

```diff
@@ -0,0 +1,42 @@
+# Default static content to be cached for 1 day
+expires 1d;
+
+if ($request_method !~ ^(GET|HEAD|OPTIONS)$) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 405;
+}
+
+if ($request_method = OPTIONS) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 204;
+}
+
+# Prefer the static files before proxying to the app
+try_files $uri @rails;
+
+location = /robots.txt  {
+  access_log off;
+  log_not_found off;
+}
+
+location = /favicon.ico  {
+  access_log off;
+  log_not_found off;
+}
+
+# Serve opensearch.xml with special content type
+location = /opensearch.xml {
+  more_set_headers 'Content-Type: application/opensearchdescription+xml';
+}
+
+# Cache assets for 1 year
+location /assets/ {
+  expires 1y;
+}
+
+# Deny access to paths with leading dots
+location ~ /\. {
+  deny all;
+  access_log off;
+  log_not_found off;
```

Comment:
We IMO want to see probe requests. All of them. To make sure we do not loose important information on logsurfing.